src/docgen/view/documentation.ts

```diff
@@ -114,29 +115,28 @@ export class Documentation {
 
       const name = options?.name ?? extractPackageName(target);
 
-      // npm7 is needed so that we also install peerDependencies - they are needed to construct
-      // the full type system.
-      console.log('Installing npm7...');
-      await spawn('npm', ['install', 'npm@7'], {
-        cwd: workdir,
-        shell: true,
-        stdio: ['ignore', 'inherit', 'inherit'],
-      });
-
       console.log(`Installing package ${target}`);
-      await spawn(path.join(workdir, 'node_modules', '.bin', 'npm'), [
+      await forkNode(require.resolve(`npm/${npmBin.npm}`), [
         'install',
         // this is critical from a security perspective to prevent
         // code execution as part of the install command using npm hooks. (e.g postInstall)
         '--ignore-scripts',
         // ensures npm does not insert anything in $PATH
         '--no-bin-links',
+        // Ignore a class of dependency conflicts
+        '--force',
+        // Include dev & peer dependencies,
+        '--include', 'dev',
```

Comment:
Why are we including dev deps? We shouldn't need to

Reply:
Almost none of the CDK documentation transliterates cleanly without it's dev dependencies... Think about example code that uses `devDependencies`... to demonstrate proper use.

Reply:
I see. I was thinking this is for generating API references but I understand why this is needed for sample code transliteration.